package.json

```
@@ -29,8 +29,8 @@
   },
   "homepage": "https://github.com/pkgjs/wiby#readme",
   "dependencies": {
-    "@octokit/graphql": "^4.5.0",
-    "@octokit/rest": "^18.0.0",
+    "@octokit/graphql": "^8.0.0",
```

Same here, the package became ESM-only.

```suggestion
    "@octokit/graphql": "^7.0.0",
```